Source/Assets/AppCenterEditorExtensions/Editor/Scripts/Utils/AppCenterEditorHelper.cs

```diff
@@ -1,20 +1,22 @@
 ﻿using UnityEditor;
 using UnityEngine;
+using System;
 using System.IO;
 
 namespace AppCenterEditor
 {
     [InitializeOnLoad]
     public static partial class AppCenterEditorHelper
     {
-        public static string EDEX_NAME = "AppCenter_EditorExtensions";
-        public static string EDEX_ROOT = Application.dataPath + "/AppCenterEditorExtensions/Editor";
+        public static string EDEX_NAME = "AppCenterEditorExtensions";
+        public static string EDEX_ROOT = Path.Combine(Application.dataPath, "AppCenterEditorExtensions/Editor");
         public static string APPCENTER_SETTINGS_TYPENAME = "AppCenterSettings";
         public static string APPCENTER_WRAPPER_SDK_TYPENAME = "WrapperSdk";
         public static string DEFAULT_SDK_LOCATION = "Assets/AppCenter";
         public static string DEFAULT_SDK_LOCATION_PATH = Application.dataPath + "/AppCenter";
         public static string MSG_SPIN_BLOCK = "{\"useSpinner\":true, \"blockUi\":true }";
         public static string ANALYTICS_SDK_DOWNLOAD_PATH = "/Resources/AppCenterAnalyticsUnitySdk.unitypackage";
+        public static string AUTH_SDK_DOWNLOAD_PATH = "/Resources/AppCenterAuthUnitySdk.unitypackage";
```

Comment:
This code should be removed because of the  App Center Auth is retired and has been removed from the SDK.

Reply:
Thanks, removed it.